OWML.ModHelper/ModCombination.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+	public class ModCombination : IModCombination
+	{
+		private bool _pressed = false, _first = false;
+		private float _firstPressed = 0f, _lastPressed = 0f;
+		private List<KeyCode> _singles = new List<KeyCode>();
+		internal string Combo { get; private set; }
+
+		public ModCombination(string combination)
+		{
+			Combo = combination;
+		}
+
+		public float GetLastPressedMoment() { return _lastPressed; }
+
+		public float GetPressDuration() { return _lastPressed - _firstPressed; }
+
+		public bool IsFirst(bool keep = false)
+		{
+			if (_first)
+			{
+				_first = keep;
+				return true;
+			}
+			return false;
+		}
+
+		internal void SetPressed(bool state = true)
+		{
+			if (state)
+			{
+				if (!_pressed)
+				{
+					_first = true;
+					_firstPressed = Time.realtimeSinceStartup;
+				}
+				_lastPressed = Time.realtimeSinceStartup;
+			}
+			else
+				_first = true;
```

Comment:
Use brackets even for single line blocks, helps with readability.

Reply:
done